index.html

```diff
@@ -8,135 +8,172 @@
     <link rel="stylesheet" href="styles/index.css" />
   </head>
 
-  <body class="app-grid">
-    <header class="app-grid__header">
+  <body>
+    <header class="header">
       <h1>Quiz App</h1>
     </header>
 
-    <main class="app-grid__main">
-      <section class="app-grid__main__card">
+    <main class="app-grid">
+      <section class="app-grid__card">
+        <img
+          src="img/icons/bookmark-regular.svg"
+          alt="bookmark"
+          class="app-grid__card--icon"
+        />
         <h2>What's your name?</h2>
-        <details class="app-grid__main__card__show-answer">
-          <summary class="app-grid__main__card__show-answer--toggle-show">
+        <details class="app-grid__card__show-answer">
+          <summary class="app-grid__card__show-answer--toggle-show">
```

Comment:
This class name is a bit confusing. With BEM a class name with `--something` indicates that this is a modifier of an existing style. For example a modifier for your bookmark class would be: `card__bookmark--active`. This `summary` element will probably have it's own style.  
```suggestion
          <summary class="card__show-answer">
```